tests/lib/prepare.sh

```diff
@@ -471,6 +480,9 @@ EOF
     # - make sure the group matches
     # - bind mount /root/test-etc/* to /etc/* via custom systemd job
     # We also create /var/lib/extrausers/* and append ubuntu,test there
+    ! test -e /mnt/system-data/root
+    mkdir -m 700 -p /mnt/system-data/root
```

Comment:
Shellcheck doesn't like (IMO it's overly picky here):
```
In tests/lib/prepare.sh line 484:
    mkdir -m 700 -p /mnt/system-data/root
          ^-- SC2174: When used with -p, -m only applies to the deepest directory.
```
Either split it or disable the warning.

Reply:
Yeah, I noticed. I think this will pass if we just say `install -m 700 -d /mnt/system-data/root`.

Reply:
Fixed.